interfaces/xenmgr_vm.xml

```diff
@@ -71,10 +72,10 @@
       </tp:docstring>
     </property>
     <property name="initrd" type="s" access="readwrite"/>
-    <property name="acpi-pt" type="b" access="readwrite">
+    <property name="acpi-path" type="s" access="readwrite">
```

Comment:
Chris explained it to me - it matches the acpi/smbios params in upstream xl (which I think I was the author of :) - good to go...